services/movie_session.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime
+from django.db.models import QuerySet
+
+
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=Movie.objects.get(id=movie_id),
+        cinema_hall=CinemaHall.objects.get(id=cinema_hall_id)
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    movies = MovieSession.objects
+    if session_date:
+        movies = movies.filter(
+            show_time__date=datetime.strptime(session_date, "%Y-%m-%d")
+        )
+    return movies.all()
```

Comment:
Why did you decide to cal .`all()` at the end ? And actually you do not need to override your viriable . You can just filter your existing query : )